### PR TITLE
Add some missing picker accessibility IDs

### PR DESCRIPTION
**Pull request by mpangburn**
For @jh-bate

### CODE REVIEW

File: LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift [hunk 1]
```diff
@@ -212,6 +213,15 @@ public struct CorrectionRangeOverridesEditor: View {
             dismiss()
         }
     }
+
+    private func accessibilityIdentifier(for preset: CorrectionRangeOverrides.Preset) -> String {
+        switch preset {
+        case .preMeal:
+            return "pre-meal"
```
Comment (jh-bate) [inline LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift:220]: so will have as `pre-meal_picker` and `workout_picker`

**Review by jh-bate — APPROVED** [review]
LGTM - I can bring the change in as part of my changes next sprint if you want?

File: LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift (hunk 1 above)
Comment (mpangburn) [reply LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift:220]: Yep!

**Comment by mpangburn** [comment]
@jh-bate I'm going to go ahead and merge this now so it's available to you whenever you need it

**Comment by ps2** [comment]
@mpangburn and @jh-bate, can you provide a link to the Jira ticket that this work was done for?

**Pull request merged**